src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The way you're setting the active stars won't work as expected. The `:nth-child(-n + 4)` selector will always select the first 4 stars regardless of the rating. You should create additional classes like `.stars--1`, `.stars--2`, etc., each one setting the correct number of active stars. For example, `.stars--1 .stars__star:nth-child(-n + 1)` would set the first star as active.